tst/worksheet.tst

```diff
@@ -0,0 +1,62 @@
+
+gap> START_TEST( "AutoDoc package: worksheet.tst" );
+gap> tmpdir := DirectoryTemporary();;
+gap> SetInfoLevel( InfoGAPDoc, 0 );
+gap> AutoDocWorksheet( "tst/worksheet.tst", rec( dir := tmpdir ) );
+gap> chap1 := Filename( Directory(tmpdir), "chap1.html" );;
+gap> IsReadableFile(chap1);
+true
+gap> ReadAll(InputTextFile(chap1));
+"<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<!DOCTYPE html PUBLIC \"-//W3C/\
+/DTD XHTML 1.0 Strict//EN\"\n         \"http://www.w3.org/TR/xhtml1/DTD/xhtml1\
+-strict.dtd\">\n\n<html xmlns=\"http://www.w3.org/1999/xhtml\" xml:lang=\"en\"\
+>\n<head>\n<title>GAP (Plain_Text_Mode_Test) - Chapter 1: Test</title>\n<meta \
+http-equiv=\"content-type\" content=\"text/html; charset=UTF-8\" />\n<meta nam\
+e=\"generator\" content=\"GAPDoc2HTML\" />\n<link rel=\"stylesheet\" type=\"te\
+xt/css\" href=\"manual.css\" />\n<script src=\"manual.js\" type=\"text/javascr\
+ipt\"></script>\n<script type=\"text/javascript\">overwriteStyle();</script>\n\
+</head>\n<body class=\"chap1\"  onload=\"jscontent()\">\n\n\n<div class=\"chli\
+nktop\"><span class=\"chlink1\">Goto Chapter: </span><a href=\"chap0.html\">To\
+p</a>  <a href=\"chap1.html\">1</a>  </div>\n\n<div class=\"chlinkprevnexttop\
+\">&nbsp;<a href=\"chap0.html\">[Top of Book]</a>&nbsp;  <a href=\"chap0.html#\
+contents\">[Contents]</a>&nbsp;  &nbsp;<a href=\"chap0.html\">[Previous Chapte\
+r]</a>&nbsp;  </div>\n\n<p id=\"mathjaxlink\" class=\"pcenter\"><a href=\"chap\
+1_mj.html\">[MathJax on]</a></p>\n<p><a id=\"X87712F9D8732193C\" name=\"X87712\
+F9D8732193C\"></a></p>\n<div class=\"ChapSects\"><a href=\"chap1.html#X87712F9\
+D8732193C\">1 <span class=\"Heading\">Test</span></a>\n</div>\n\n<h3>1 <span c\
+lass=\"Heading\">Test</span></h3>\n\n<p>This is dummy text</p>\n\n\n<div class\
+=\"example\"><pre>\n<span class=\"GAPprompt\">gap&gt;</span> <span class=\"GAP\
+input\">S5 := SymmetricGroup(5);</span>\nSym( [ 1 .. 5 ] )\n<span class=\"GAPp\
+rompt\">gap&gt;</span> <span class=\"GAPinput\">Size(S5);</span>\n120\n</pre><\
+/div>\n\n<p>And we wrap up with some dummy text</p>\n\n\n<div class=\"chlinkpr\
+evnextbot\">&nbsp;<a href=\"chap0.html\">[Top of Book]</a>&nbsp;  <a href=\"ch\
+ap0.html#contents\">[Contents]</a>&nbsp;  &nbsp;<a href=\"chap0.html\">[Previo\
+us Chapter]</a>&nbsp;  </div>\n\n\n<div class=\"chlinkbot\"><span class=\"chli\
+nk1\">Goto Chapter: </span><a href=\"chap0.html\">Top</a>  <a href=\"chap1.htm\
+l\">1</a>  </div>\n\n<hr />\n<p class=\"foot\">generated by <a href=\"http://w\
+ww.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc\">GAPDoc2HTML</a></p>\n</body>\n<\
+/html>\n"
+gap> STOP_TEST( "plaintextmode.tst", 10000 );
+#! @Title Plain Text Mode Test
+#! @Date 2018/08/17
+#! @Chapter Test
+#! This is dummy text
+#! @BeginExampleSession
+#! gap> S5 := SymmetricGroup(5);
+#! Sym( [ 1 .. 5 ] )
+#! gap> Size(S5);
+#! 120
+#! @EndExampleSession
+#! And we wrap up with some dummy text
```

Comment:
Actually, I'd prefer if the worksheet input was in a separate file, and not mixed up with the .tst file.